prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -55,10 +55,10 @@ class BatchFunction {
                     receiver.timing?.operation == Receiver.BatchOperation.MERGE -> listOf(Report.merge(inReports))
                     else -> inReports
                 }
-                val outReports = when (receiver.format) {
-                    Report.Format.HL7 -> mergedReports.flatMap { it.split() }
-                    else -> mergedReports
-                }
+                val outReports = if (receiver.format.isSingleItemFormat)
+                    mergedReports.flatMap { it.split() }
+                else
+                    mergedReports
```

Comment:
Single item formats are a concept in other parts of the code. This change allows new single-item formats to be defined without changing this piece of code.